pkg/sitewise/api/list_associated_assets.go

```diff
@@ -14,6 +14,33 @@ import (
 
 func ListAssociatedAssets(ctx context.Context, client client.SitewiseClient, query models.ListAssociatedAssetsQuery) (*framer.AssociatedAssets, error) {
 
+	var assetId *string = util.GetAssetId(query.BaseQuery)
```

Comment:
[style] better follow the existing coding style, i.e. make the var declaration at the top of the func and then start implementation.

Reply:
I think this is the way you suggest now, let me know if I misunderstood.